apps/chronicles/screens/components/PopoverScreen.tsx

```diff
@@ -10,37 +10,74 @@ import React, { useRef, useState } from "react";
 import { ScrollView, View } from "react-native";
 
 export const PopoverScreen = () => {
-    const buttonRef = useRef<View>(null);
-    const [open, setOpen] = useState(false);
+    const firstButtonRef = useRef<View>(null);
+    const [isFirstPopoverOpen, setIsFirstPopoverOpen] = useState(false);
+
+    const secondButtonRef = useRef<View>(null);
+    const [isSecondPopoverOpen, setIsSecondPopoverOpen] = useState(false);
+
     const style = useStyles(themeStyles);
     return (
         <ScrollView
             contentInsetAdjustmentBehavior="automatic"
             contentContainerStyle={style.contentContainer}
         >
             <Typography>
-                {
-                    "The popover component displays modally on top of your content and positions itself \
-                automatically based on constraints around the anchor element so you won't have to \
-                worry about it being positioned badly."
-                }
+                {"The popover component displays modally on top of your content and positions " +
+                    "itself automatically based on constraints around the anchor element so you " +
+                    "won't have to worry about it being positioned badly."}
             </Typography>
-            <Spacer amount="large" />
+            <Spacer />
             <Button
                 title="Trigger popover"
-                ref={buttonRef}
+                ref={firstButtonRef}
                 onPress={() => {
-                    setOpen(!open);
+                    setIsFirstPopoverOpen(!isFirstPopoverOpen);
                 }}
             />
             <Popover
-                open={open}
-                anchorEl={buttonRef}
-                onClose={() => setOpen(false)}
+                open={isFirstPopoverOpen}
+                anchorEl={firstButtonRef}
+                onClose={() => setIsFirstPopoverOpen(false)}
                 placement="top"
             >
                 <Typography>This is a popover</Typography>
             </Popover>
+
```

Comment:
Any reason to keep this empty line?

Reply:
Removed it.
For formatting comments like this, I think it is better that we make use of prettier for formatting properly.